modules/nf-core/thermorawfileparser/main.nf

```diff
@@ -29,8 +29,8 @@ process THERMORAWFILEPARSER {
 
     """
     ThermoRawFileParser.sh \\
-        --input $raw \\
-        --output_file ${prefix}.${suffix} \\
+        -i $raw \\
+        -b ${prefix}.${suffix} \\
```

Comment:
Are the long form argument gone in the new version?

Reply:
Technically not, they changed the output argument though 🙄.